themes/agnoster/agnoster.theme.sh

```diff
@@ -104,10 +104,10 @@
 
 # then add the following to your .bashrc:
 
-# export THEME=$HOME/.bash/themes/agnoster-bash/agnoster.bash
-# if [[ -f $THEME ]]; then
+# export THEME="$HOME/.bash/themes/agnoster-bash/agnoster.bash"
+# if [[ -f "$THEME" ]]; then
```

Comment:
One of the ideas to use the conditional commands `[[ ... ]]` is to make use of the fact that the arguments doesn't undergo word splitting and pathname expansions. So I would like to normalize the codebase so that we avoid the quoting of the arguments in the conditional commands (except for the right-hand side of `==`, `!=`, and `=~`, where the quoting takes effect).

```suggestion
# if [[ -f $THEME ]]; then
```